packages/core/src/Engine.ts

```
@@ -504,6 +484,9 @@
 
     Shader._clear(this);
     this._hardwareRenderer.destroy();
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)